packages/rum/src/domain/record/utils.ts

```diff
@@ -1,5 +1,8 @@
 import { assign, timeStampNow } from '@datadog/browser-core'
-import type { IncrementalData, IncrementalSnapshotRecord } from '../../types'
+import type {
+  BrowserIncrementalData as IncrementalData,
+  BrowserIncrementalSnapshotRecord as IncrementalSnapshotRecord,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Rename those types in `src/types/index.ts`, before exporting them, so we have a consistent names accross the codebase. The drawback is that you couldn't use `export * from`. WDYT?

Reply:
Well, it adds another process on top of re-generating the types.
From the player (and the product) perspective, there's nothing wrong in calling those types `Browser*`.

Reply:
> Well, it adds another process on top of re-generating the types.

I don't think so? We would change `src/types/index.ts` only when we need to, no need to be exhaustive here.

>  From the player (and the product) perspective, there's nothing wrong in calling those types Browser*.

I understand, no issue with that.

I would be OK with using `Browser*` types in the Browser SDK instead of using `import as` everytime we want to use them. It is not enforced so we need to think about it all the time, and it breaks auto-import.

Reply:
I was talking about the `index.ts` sure.

It's fine to get rid of the aliases, just that I didn't want to make all those changes in the scope of this PR.

Reply:
As agreed with @BenoitZugmeyer, I'm going to get rid of the aliases right in this PR.
Will send a follow-up commit.